tests/lib/test_snapshot.py

```diff
@@ -48,25 +48,25 @@ def decode(_snapshot):
         return snapshot.decodeSnapshot(_snapshot)
 
     encoded_snapshot = encode(block_number, value)
-    assert decode(encoded_snapshot) == [block_number, value]
+    assert decode(encoded_snapshot) == (block_number, value)
     expected_encoded_snapshot_as_bytes = block_number.to_bytes(4, "big") + value.to_bytes(12, "big")
-    assert Web3.toBytes(encoded_snapshot).rjust(16, b"\x00") == expected_encoded_snapshot_as_bytes
+    assert Web3.to_bytes(encoded_snapshot).rjust(16, b"\x00") == expected_encoded_snapshot_as_bytes
 
     # Testing adding new snapshots
     account = accounts[0]
 
     data = [(block_number + i * 10, value + i) for i in range(10)]
     for i, (block_i, value_i) in enumerate(data):
-        snapshot.addSnapshot(block_i, value_i, {"from": account})
+        snapshot.addSnapshot(block_i, value_i, sender=account)
 
         assert snapshot.length() == i + 1
-        assert snapshot.history(i) == encode(block_i, value_i)
-        assert snapshot.lastSnapshot() == [block_i, value_i]
+        assert snapshot.call_view_method("history", i) == encode(block_i, value_i)
```

Comment:
you can also call non-view methods like this in ape:
```suggestion
        assert snapshot.history.call(i) == encode(block_i, value_i)
```

Reply:
still getting error: `AttributeError: 'AccountHistory' object has no attribute 'call'`

Reply:
`snapshot._mutable_methods_["history"].call(...)` would be the way to do that when there's a conflict with an ape property

Reply:
yep, we have `history` view method, is current option with `snapshot.call_view_method("history", i)` ok?

Reply:
Yes, that works too